src/utils/theme-composer/src/index.ts

```diff
@@ -0,0 +1,113 @@
+// @ts-nocheck
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+// if we rename packages to include category we can remove these mappings
+const ELEMENTS = [
+  '@uswitch/trustyle.accordion',
+  '@uswitch/trustyle.author',
+  '@uswitch/trustyle.author-profile',
+  '@uswitch/trustyle.badge',
+  '@uswitch/trustyle.breadcrumbs',
+  '@uswitch/trustyle.button',
+  '@uswitch/trustyle.call-out',
+  '@uswitch/trustyle.card',
+  '@uswitch/trustyle.category',
+  '@uswitch/trustyle.cta',
+  '@uswitch/trustyle.funnel-progress',
+  '@uswitch/trustyle.flex-grid',
+  '@uswitch/trustyle.hero',
+  '@uswitch/trustyle.input',
+  '@uswitch/trustyle.list',
+  '@uswitch/trustyle.pagination',
+  '@uswitch/trustyle.toggle-switch'
+]
+const COMPOUNDS = [
+  '@uswitch/trustyle.article-intro',
+  '@uswitch/trustyle.side-nav'
+]
```

Comment:
Can't you look these up in the package.json files?

Reply:
there's no mapping for what's an element and what's a compound

Reply:
(they are looked up in package.json - this assigns them to the correct group)

Reply:
ahh right because they're coming from node_modules, not this project - got it.

Reply:
Could also add it as another field in package.json

Reply:
Yeah or if the package names contained it, i.e. `@uswitch/trustyle.element.input`